.prettierignore

```diff
@@ -1,2 +1,5 @@
-build
 coverage
+lib
+tmp
+
+*.yaml
```

Comment:
Need to add the whole `charts` directory here.

Reply:
Added.